aiohttp/client.py

```diff
@@ -150,6 +156,34 @@
     SSLContext = None
 
 
+class _RequestOptions(TypedDict, total=False):
+    params: Optional[Mapping[str, str]]
```

Comment:
Could you replace `Optional` with `Union` everywhere? It's considered semantically confusing.

Reply:
Fixed. https://github.com/aio-libs/aiohttp/pull/8463/commits/c111e2e3acc6490f9f0071d4181f4ddcb5348fff